src/highlight.js

```diff
@@ -707,8 +707,8 @@ https://highlightjs.org/
   }
 
   function autoDetection(name) {
-    name = (name || '').toLowerCase();
-    return !languages[name].disableAutodetect;
+    lang = getLanguage(name);
+    return lang && !lang.disableAutodetect;
```

Comment:
Does lang have to be a global or closure? Could it be a local var?

Reply:
Yep, you're right of course.